operator/v1/0000_70_cluster-network-operator_01.crd.yaml-patch

```diff
@@ -1,3 +1,18 @@
 - op: add
   path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/defaultNetwork/properties/ovnKubernetesConfig/properties/policyAuditConfig/default
   value: {}
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/exportNetworkFlows/properties/ipfix/properties/newCollectors/items/properties/address/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
```

Comment:
so why can't this be specified inline? does kubebuilder not support `format` or something?